apis/userapi/userapi.yaml

```diff
@@ -1937,78 +1950,234 @@ paths:
+                                type: state
+                              - id: 107c9472-a950-4768-bcd6-f882910177c4
+                                type: district
+                              - id: ba31e7c2-fac9-472a-8867-1582c73bcca8
+                                type: block
+                              - id: 5c8876b5-981c-4c6a-b23d-20c9e17428e1
+                                type: cluster
+                            externalId: 33100800608
+                            updatedDate: 2021-04-07 05:09:38:366+0000
+                            userId: 96102c3f-2c22-4614-8dcc-6b130cefe586
+                            approvaldate: null
+                            isSchool: true
+                            isDeleted: false
+                            hashTagId: 0127419693630996481321
+                            isRejected: false
+                            locations:
+                              - code: 33
+                                name: Tamil Nadu
+                                id: 91d9baae-14f1-477a-955c-f91bd9037f0b
+                                type: state
+                                parentId: null
+                              - code: 3310
+                                name: ERODE
+                                id: 107c9472-a950-4768-bcd6-f882910177c4
+                                type: district
+                                parentId: 91d9baae-14f1-477a-955c-f91bd9037f0b
+                              - code: 33100169
+                                name: Gobi
+                                id: ba31e7c2-fac9-472a-8867-1582c73bcca8
+                                type: block
+                                parentId: 107c9472-a950-4768-bcd6-f882910177c4
+                              - code: 3310016901
+                                name: Gobi Cluster
+                                id: 5c8876b5-981c-4c6a-b23d-20c9e17428e1
+                                type: cluster
+                                parentId: ba31e7c2-fac9-472a-8867-1582c73bcca8
+                            id: 0132527931586969602
+                            position: null
+                            isApproved: false
+                            orgjoindate: 2021-04-07 05:09:45:655+0000
+                            orgLeftDate: 2021-04-07 05:09:38:366+0000
+                        provider: null
+                        countryCode: null
+                        tncLatestVersionUrl: https://sunbirdstagingpublic.blob.core.windows.net/termsandcondtions/terms-and-conditions-v9.html#termsOfUse
+                        maskedEmail: k1**@yopmail.com
+                        email: k1**@yopmail.com
+                        rootOrg:
+                          dateTime: null
+                          preferredLanguage: null
+                          keys:
+                          approvedBy: null
+                          channel: dikshapreprodcustodian
```

Comment:
@sreeragksgh @mohitga please remove all references to diksha and any specific servers unless necessary from Sunbird documentation.